cleaner/environment_matrix/classes/clean.php

```diff
@@ -96,7 +96,14 @@ static public function execute() {
                         }
 
                         if (!$dryrun) {
-                            set_config($config->config, $config->value, $config->plugin);
+
+                            $classname = $config->plugin.'\admin\setting_'.$config->config;
```

Comment:
This doesn't look right, isn't ->config they config item key name?

This needs to map somehow to the config type, which is not dependant on either the config plugin or config name.

eg lets say we have tool_crawler/botpassword => foobar

We need the $classname to be 'admin_setting_configpasswordunmask'